filesystem/config.pan

```diff
@@ -5,8 +5,28 @@ unique template filesystem/config;
 
 include { 'quattor/functions/filesystem' };
 
+
+@{
+desc =  template customizing the default disk layout provided by the default configuration
+values = template namespace
+default = null
+required = no
+}
 variable FILESYSTEM_LAYOUT_CONFIG_SITE ?= null;
 
+@{
+desc =  template included at the beginning of the file system configuration and allowing to \
+ to redefine the default size of block devices defined in the default configuration
+values = template namespace
+default = FILESYSTEM_LAYOUT_CONFIG_SITE+'-init' or null if FILESYSTEM_LAYOUT_CONFIG_SITE is undefined
+required = no
+}
+variable FILESYSTEM_LAYOUT_CONFIG_INIT ?= if ( is_defined(FILESYSTEM_LAYOUT_CONFIG_SITE) ) {
+                                            if_exists(FILESYSTEM_LAYOUT_CONFIG_SITE + '-init');
+                                          } else {
+                                            null;
+                                          };
```

Comment:
I don't really understand your point... This is the standard formatting used through the template library when the `if` follows the equal sign...